src/Stratis.CirrusPegD/Program.cs

```diff
@@ -62,8 +78,12 @@ private static IFullNode GetMainchainFullNode(string[] args)
                 MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
             };
 
+            var fedPegOptions = new FederatedPegOptions(
```

Comment:
Can `FederatedPegOptions` be renamed to `FederatedPegSettings` to align with the naming of the other settings classes?

Reply:
We already have FederationGatewaySettings, which is aligned with what NodeSettings, MinerSettings et al are doing... It's hard to put this in there though because we have to inject the correct network.

This is more a kind of option to be passed to the node builder.

Reply:
I also think its not settings (settings is more related to configuration of the node), maybe options is also not the best name but its reasonably fine. An alternative name would be CrossNetworkMappings or so, but Options is good enough and we are pressed in time.

Reply:
A setting is anything we would want to configure or override from the command line now or perhaps in the future that modifies the behavior of a feature.